docs/tutorial.rst

```diff
@@ -502,6 +502,12 @@ result is automatically converted to a string. For example,
     42 yo
     >>> echo "hello" | @(lambda a, s=None: s.read().strip() + " world\n")
     hello world
+    >>> @(['echo', 'hello world'])
+    hello world
```

Comment:
I don't understand, I guess my point was that @('echo hello world') would raise an error, while does not.

Reply:
That's true, but I think having the space in the "hello world" string adds confusion because `@(['echo', 'hello world'])` and `@(['echo', 'hello', 'world'])` produce the same output

Reply:
sorry, my explanation was bad :)

Reply:
Do you want the raise in the example?

Reply:
The raise?